velox/docs/functions/spark/string.rst

```diff
@@ -179,6 +179,18 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT substring('Spark SQL', -10, 3); -- "Sp"
         SELECT substring('Spark SQL', -20, 3); -- ""
 
+.. spark:function:: substring_index(string, delim, count) -> varchar
+
+    Returns the substring from ``string`` before ``count`` occurrences of the delimiter ``delim``.
+    If ``count`` is positive, everything to the left of the final delimiter
+    (counting from the left) is returned. If ``count`` is negative, everything to the right
+    of the final delimiter (counting from the right) is returned. The function
+    substring_index performs a case-sensitive match when searching for ``delim``. ::
```

Comment:
Drop "The function substring_index...": "delim is case-sensitive."

Can delim be an empty string? What happens in that case?

Reply:
If the delim is empty. It will return empty string.